tests/005-test-devmapper-cleanup

```diff
@@ -0,0 +1,61 @@
+source $SRCDIR/libtest.sh
+
+cleanup_local() {
+  local test_status
+  local vg_name=$1
+  local devs=$2
+  $DSSBIN --reset >> $LOGS 2>&1
+  test_status=0
+  if [ -e /etc/sysconfig/docker-storage ]; then
+     echo "  $DSSBIN reset Failed: $(basename testname)"
+     test_status=1
+  fi
+  cleanup $vg_name $devs
```

Comment:
Can we move this check outside of cleanup_local(). In fact we can get rid of cleanup_local() completely and keep everything in parent function, That's much more readable. This function is doing too much. Calling --reset, settin test status as well as doing actual cleanup. 

I think we can easily do that from parent function and that's more readable.